chaos_genius/connectors/base_db.py

```diff
@@ -1,6 +1,10 @@
+import logging
+
 from sqlalchemy import text
 from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import text, exc as sqlalchemy_exc
```

Comment:
`text` is imported twice

Reply:
According to `isort` it should be:

```suggestion
from flask_sqlalchemy import SQLAlchemy
from sqlalchemy import exc as sqlalchemy_exc
from sqlalchemy import text
```